pycbc/inference/models/relbin.py

```diff
@@ -97,9 +97,21 @@ def setup_bins(f_full, f_lo, f_hi, chi=1.0, eps=0.5, gammas=None):
     # frequency grid points
     fbin = dphi2f(dphi_grid)
     # indices of frequency grid points in the FFT array
-    fbin_ind = numpy.unique(
-        [numpy.argmin(numpy.absolute(f_full - ff)) for ff in fbin]
-    )
+    fbin_ind = numpy.searchsorted(f_full, fbin)
+    for idx_fbin, idx_f_full in enumerate(fbin_ind):
```

Comment:
I wanted to exactly reproduce what was done before. 

`searchsorted([1,2,3,4], 1.2)` would return [1] (*effectively* rounding up), whereas the previous code would want [0] as 1 is closer than 2. If it doesn't matter how this is handled, then I'm happy to cut this, but I think it would be better to have an optimization patch that changes nothing and *then* a simplification patch, rather than have both in one.

Reply:
I think it does need to go to nearest actually. There is a simpler way to do that, but as you say it can be simplified in a followup. I just wanted to make sure I understood the behavior of the lines below. Otherwise, this is fine.